docs/source/reference/config.rst

```diff
@@ -113,3 +113,33 @@ Available fields and semantics:
       # Only one element is allowed in this list, as GCP disallows multiple
       # specific_reservations in a single request.
       - projects/my-project/reservations/my-reservation
+
+  # Advanced Kubernetes configurations (optional).
+  kubernetes:
+    # The networking mode for accessing jump pods (optional).
+    # This must be either: 'nodeport' or 'portforward'.
+    #
+    # nodeport: Exposes a service on a static port number on each Node, allowing external access to services using <NodeIP>:<NodePort>.
+    # portforward: Forward a local port to a port on a Pod, typically used for debugging or temporary access purposes.
+    networking:  nodeport
```

Comment:
There seem to be two spaces here :) Also, should we default to portforward here?
```suggestion
    networking: portforward
```